plugins/follow_me/follow_me_impl.cpp

```diff
@@ -149,13 +144,9 @@ FollowMe::Result FollowMeImpl::stop()
     }
     // Note: the safety flag is not needed in future versions of the PX4 Firmware
```

Comment:
I think you can remove this comment everywhere it appeared other than in `set_flight_mode()`.